assets/js/base/components/checkbox-control/index.js

```diff
@@ -35,6 +35,15 @@ const CheckboxControl = ( {
 				onChange={ ( event ) => onChange( event.target.checked ) }
 				{ ...rest }
 			/>
+			<svg
+				className="wc-block-checkbox__mark"
+				aria-hidden="true"
+				xmlns="http://www.w3.org/2000/svg"
+				viewBox="0 0 24 24"
+			>
+				<path d="M0 0h24v24H0z" fill="none" />
```

Comment:
Could this line be removed?
```suggestion
```

Everything seems to look good for me after removing it:
![imatge](https://user-images.githubusercontent.com/3616980/80391792-82e5b280-88ae-11ea-85b2-1fe8483ffcc5.png)

Reply:
you mean it fixes the IE11 issue?

Reply:
Ah no, it's not related. I think that `<path>` can be removed because it has `fill="none"`, I couldn't see any difference after removing it.